client/lib/plans/features-list.js

```diff
@@ -289,10 +289,17 @@ export const FEATURES_LIST = {
 	[ FEATURE_ADVANCED_DESIGN_CUSTOMIZATION ]: {
 		getSlug: () => FEATURE_ADVANCED_DESIGN_CUSTOMIZATION,
 		getTitle: () => i18n.translate( 'Advanced design customization' ),
-		getDescription: () =>
-			i18n.translate(
+		getDescription: () => {
+			return i18n.hasTranslation(
 				"Access extended color schemes and customization tools. Change all of your site's fonts, colors and more."
-			),
+			)
+				? i18n.translate(
+						"Access extended color schemes and customization tools. Change all of your site's fonts, colors and more."
+				  )
+				: i18n.translate(
+						'Access extended color schemes, backgrounds, and CSS, giving you complete control over how your site looks.'
```

Comment:
Since `FEATURE_ADVANCED_DESIGN_CUSTOMIZATION` is available on the Personal plan, this can make users think that the ability to edit CSS is included when the translation is missing.

Reply:
Good point and that makes things a bit complicated :).